torch/nn/modules/flatten.py

```diff
@@ -95,14 +95,22 @@ class Unflatten(Module):
 
     __constants__ = ['dim', 'unflattened_size']
     dim: Union[int, str]
-    unflattened_size: Union[Size, NamedShape]
+    unflattened_size: Union[Size, List, NamedShape]
 
-    def __init__(self, dim: Union[int, str], unflattened_size: Union[Size, NamedShape]) -> None:
+    def __init__(self, dim: Union[int, str], unflattened_size: Union[Size, List, NamedShape]) -> None:
```

Comment:
`_size` is a better annotation for `unflattened_size` now. See here:
 https://github.com/pytorch/pytorch/blob/963f7629b591dc9750476faf1513bc7f1fb4d6de/torch/types.py#L22